content/en/quickstart/quickstart-cosign.md

```diff
@@ -1,22 +1,26 @@
 ---
 type: docs
-category: Signing
-description: ''
-menuTitle: Quick Start
-title: Cosign
-weight: 100
+category: Quickstart
+description: Sign and Verify with Cosign
+title: Sigstore Quickstart with Cosign
+weight: 5
 ---
 
 ![Cosign Overview](/sigstore_cosign-horizontal-color.svg)
 
-## Getting Started (Quick Start)
-
-Cosign is a command line utility that can sign and verify software artifact, such as container images and blobs.
 Join us on our [Slack channel](https://sigstore.slack.com/). (Need an [invite](https://links.sigstore.dev/slack-invite)?)
 
+## Quickstart signing and verifying with Cosign
+
+Cosign is a command line utility that is used to sign software artifacts and verify signatures using Sigstore.
+
+Language specific clients (like [sigstore-python](https://github.com/sigstore/sigstore-python)) are other options for signing and verifying, but Cosign is a great, language agnostic place to start. 
+
+This quickstart will walk you through how to sign and verify a blob and a container. Although keyless signing is recommended, this quickstart will also show you how to sign using a generated key. 
```

Comment:
Maybe this is out of scope for this reorg PR, but since we're recommending keyless signing maybe that's what the focus of this quickstart should be?

Reply:
@cmurphy So it does go through keyless signing first, and then signing with a generated key at the end. Would you like for me to remove the generated key section? 

I think I could reword things so that it is clearer that the first part of the quickstart is for keyless signing and is preferred. However, a quickstart does not need to highlight every feature so I am comfortable removing the generated key section all together.

Reply:
I think I was initially thinking to just remove the non-keyless signing part since I imagine the quickstart as a very brief, easy introduction, but I'd also be fine with just rewording this to de-emphasize signing with a generated key. If we go with the latter, either removing the second sentence entirely or changing it to something like

```suggestion
This quickstart will walk you through how to sign and verify a blob and a container. It will introduce keyless signing, which is the recommended method, as well as signing with a generated key, which is necessary for some use cases.
```

I think it's mainly starting the sentence with "Although" that's throwing me.

Reply:
I think that there is a lot going on in this quickstart, so I opted to remove the section. Let me know what you think.